### PR TITLE
fix commen typo

### PR DESCRIPTION
**Pull request by linzhaoming**
fix commen typo

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [x] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

# Release Notes

### CODE REVIEW

File: pkg/apis/pipeline/images.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ type Images struct {
 	KubeconfigWriterImage string
 	// ShellImage is the container image containing bash shell.
 	ShellImage string
-	// GsutilImage is the container miage containing gsutil.
+	// GsutilImage is the container  containing gsutil.
```
Comment (pritidesai) [inline pkg/apis/pipeline/images.go:39]: great catch @linzhaoming, can we fix the typo and have the comment with "image"?

Comment (linzhaoming) [reply pkg/apis/pipeline/images.go]: done

Comment (pritidesai) [reply pkg/apis/pipeline/images.go]: thanks

**Review by pritidesai — APPROVED** [review]
@linzhaoming, Welcome to the community! 

🎉 Thank you for your contribution 🎉

**Comment by jerop** [comment]
welcome @linzhaoming 🥳 

/lgtm
/meow

**Pull request merged**